src/components/PaymetsTable/PaymentsTable.js

```diff
@@ -17,23 +17,25 @@ const PaymentsTable = ({ paymentsData }) => {
   };
 
   const headCells = [
-    { id: 'id', numeric: true, disablePadding: true, label: 'ID' },
     {
-      id: 'Student',
+      id: 'id',
+      numeric: true,
+      disablePadding: true,
+      label: 'ID',
+    },
+    {
+      id: 'student',
       numeric: false,
       disablePadding: true,
       label: 'ФИО',
-      renderCell: function editBtn(row) {
-        return `${row.Student.lastName} ${row.Student.firstName}`;
-      },
     },
 
     {
       id: 'date',
       numeric: false,
       disablePadding: true,
       label: 'Дата платежа',
-      renderCell: function editBtn(row) {
+      renderCell: function date(row) {
         return format(new Date(row.date), 'MM/dd/yyyy');
```

Comment:
Написали американский формат даты, поменяй пожалуйста на привычный для нас формат: день.месяц.год 
```suggestion
        return format(new Date(row.date), 'dd.MM.yyyy');
```

Reply:
Такой формат применен везде для консистентности отображения даты. 
Если поменять здесь, то надо поменять в формах. Но в формах нельзя поменять, т.к. material-ui выдает ошибку: The specified value "29.07.2021" does not conform to the required format, "yyyy-MM-dd".

![image](https://user-images.githubusercontent.com/45433350/127444092-df9b73c6-5f0e-4d9e-b655-6f3d50f6e5f3.png)